config/routes.rb

```diff
@@ -1,4 +1,9 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  root 'task#index'
+  get 'task/index'
```

Comment:
if `root` already defines same route you don't need to set it again.

You should add it as `get 'tasks', to: 'tasks#index` to cover case where users would go to '/tasks` to see all the tasks - that's how we do it in Foreman and it's good practice in general